### DISCUSSION TITLE
V4 firmware install using ESPSFlashTool.jar fails

### DISCUSSION THREAD

**Discussion by pbernhard** [forkineye/ESPixelStick#585] [Q&A] [answered]
While attempt to install V4 firmware to a ESPixelStick V3 with the ESP8266 MCU fails when executing the ESPSFlashTool.jar in Windows 10 and produces the following Message Box :

![image](https://user-images.githubusercontent.com/6126770/203122895-d64ac1e7-8fa2-4132-a450-8b814d3e4a2b.png)

Note: using the ESPSFlashTool.jar provided with the V3.2 repository is successful.

It appears the esp8266 bin files are indeed missing from the esptool.   Is there a solution to enable the ESPSFlashTool.jar to function as expected for V4 firmware?

**Comment by MartinMueller2003** [comment, marked as answer]
Try this flash tool https://drive.google.com/file/d/1DG3wz8lbYNc371OIbeOqkzMaY5Zd2SKA/view?usp=sharing

**Reply by pbernhard** [reply]
Thank you, Martin, this install tool worked as expected.  Installation successful.   Thank you for your quick response.
